draft-ietf-quic-multipath.md

```diff
@@ -127,12 +127,37 @@ path management in {{Section 9 of QUIC-TRANSPORT}} and therefore
 requires negotiation between the two endpoints using a new transport
 parameter, as specified in {{nego}}.
 
-This extension uses multiple packet number spaces.
-When multipath is negotiated, each separate packet number space is linked to a path ID.
+This extension specifies a new Path Identifier (Path ID), which is an
+integer between 0 and 2^32 - 1 (inclusive). Path identifies are generated
+monotonically increasing and cannot be reused.
+
+The Path ID is used to
+address a path the new multipath control frames,
+such as PATH_ABANDON {{path-abandon-frame}}, PATH_STANDBY {{path-standby-frame}}},
+PATH_AVAILABLE {{path-available-frame}} as well as ACK_MP {{ack-mp-frame}}.
+Further, Connection IDs are issued per Path ID.
+Each connection ID is associated with one path identifier
+but multiple connection IDs can be associated with the same path identifier.
+
+The Path ID of the initial path is 0. Connection IDs
+which are issued by an NEW_CONNECTION_ID frame {{Section 19.15. of QUIC-TRANSPORT}}
+respectively are asociated with Path ID 0. Also, the Path ID for
+the connection ID specified in the "preferred address" transport parameter is 0.
+Use of the "preferred address" is considered as a migration event
+that does not change the Path ID.
+
+This extension uses multiple packet number spaces, one for each active path.
+As such each path has two associated packet number spaces for each direction.
```

Comment:
I think I understand the meaning here, but the definition of "packet number space" in the first sentence (one per path) is not coherent with the second one (one per direction over a path). Here is a first try to better describe this, but I think we could do better.

```suggestion
This extension uses multiple packet number spaces, one for each active path.
As such, each path maintains distinct packet number states for sending and receiving packets, as in {{QUIC-TRANSPORT}}.
```